README.md

```diff
@@ -460,11 +460,12 @@ These are hour-long Crowdcast events with a panel of speakers that will discuss
 |    Session   |   Speakers|    Description                                                            |
 |-------------------|--------------------|-----------------------------------------------------------------------|
 |  **3.1 Open Science**  |  Nici Pfeiffer (COS), Mark Musen (CEDAR, Stanford)   | The Center for Open Science develops tools for project management, collaboration, preregistration, and data sharing. CEDAR enables investigators to author metadata, annotating experimental datasets to make data more findable, accessible, interoperable, and reusable (FAIR).      
-|   **7.1 Job fair**  |  Facebook Reality Labs | Research careers at FRL, a  division of Facebook focused on creating the future of virtual and augmented reality. 
-|   **7.2 Job fair**  |  Facebook Reality Labs | Research careers at FRL, a  division of Facebook focused on creating the future of virtual and augmented reality. 
-|   **8.1 Careers in industry**  | Becky Clarkson (Apple), Kyle Frankovich (Insight Data Science), Kachi Odoememe (APL) | Research careers in industry. 
-|   **8.2 Careers in industry**  | Feryal Behbahani (DeepMind), Sophie Kenny (Vpixx), Graeme Moffat (System2 Neurotechnology) | Research careers in industry.
-|   **9.1 Careers in academia**  | Sripati Panditaradhyula Arun (IISc), Minhae Kwon (Soongsil University), Taro Toyoizumi (RIKEN) | Research careers in academia.
+|   **7.1 Job Fair**  |  Facebook Reality Labs | Research careers at FRL, a  division of Facebook focused on creating the future of virtual and augmented reality. 
+|   **7.2 Job Fair**  |  Facebook Reality Labs | Research careers at FRL, a  division of Facebook focused on creating the future of virtual and augmented reality. 
+|   **8.1 Careers in Industry**  | Becky Clarkson (Apple), Kyle Frankovich (Insight Data Science), Kachi Odoememe (APL) | Research careers in industry. 
+|   **8.2 Careers in Industry**  | Feryal Behbahani (DeepMind), Sophie Kenny (Vpixx), Graeme Moffat (System2 Neurotechnology) | Research careers in industry.
+|   **9.1 Careers in Academia**  | Sripati Panditaradhyula Arun (IISc), Minhae Kwon (Soongsil University), Taro Toyoizumi (RIKEN) | Research careers in academia.
+|   **10.1 How We Built This**  | Patrick Mineault, Tara van Viegen, Michael Waskom, Madineh Sarvcestani| 200 TAs, 2000 students, 20 days of content, 20 hours of video, 200 volunteers - how we built and launched NMA in 2 months
```

Comment:
Love the "2" repetition but would it be better to go 2000, 200,  20.... ? Not super important so I will mark as approved and leave up to you.

Reply:
Agreed! I changed it.